src/Service/MessageTranslator.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service;
+
+use Yiisoft\I18n\MessageReaderInterface;
+
+final class MessageTranslator implements MessageReaderInterface
```

Comment:
Why introducing it if we have https://github.com/yiisoft/i18n-message-php?

Reply:
fixed.